hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManagerStarter.java

```diff
@@ -57,6 +58,9 @@ public class StorageContainerManagerStarter extends GenericCli {
       LoggerFactory.getLogger(StorageContainerManagerStarter.class);
 
   public static void main(String[] args) {
+    // redirect JUL to slf4j
+    SLF4JBridgeHandler.removeHandlersForRootLogger();
+    SLF4JBridgeHandler.install();
```

Comment:
Should we move this in `GenericCli`, to apply to all Ozone components (client and server)?